cypress/support/pageobjects/Homepage.js

```diff
@@ -63,15 +63,15 @@ module.exports = class HomePage {
 	}
 
 	getFindEventLink() {
-		return cy.contains("Find Events");
+		return cy.get(":nth-child(1) > .call-to-action__action > a");
```

Comment:
This is asking to break in the future - any way of getting a more generic selector for it?

Reply:
would `cy.contains("Go to an event")` work here @skumarqa2010?

Reply:
Yaah, sorry got confused  as**"Find Events"** got charged to **"Find events"** so earlier selector was not working.